lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java

```diff
@@ -121,6 +121,10 @@ public void setDelegate(@Nullable ImageAssetDelegate assetDelegate) {
       Logger.warning("Unable to decode image.", e);
       return null;
     }
+    if (bitmap == null) {
+        Logger.warning("Decoded image is NULL.");
```

Comment:
```suggestion
        Logger.warning("Decoded image (${id}) is null.");
```

Do you mind changing the `Unable to decode image.` above to `Unable to decode image(${id}).` as well? I think it will be more informative.

Reply:
Very good advice. But this is Java not Kotlin, so I made a little change.